rootmap/errors.go

```diff
@@ -20,3 +20,20 @@ func (e TableNotFoundError) Is(target error) bool {
 	_, ok := target.(TableNotFoundError)
 	return ok
 }
+
+type RequestedRootTruncatedError struct {
```

Comment:
let's stick a docstring on here

Reply:
We didn't end up using this! Instead of erroring, I leave it up to the caller of the rootmap function to decide how to deal with the root version being <= the truncated version

Reply:
Removing this error